varnish/metadata.csv

```diff
@@ -240,27 +312,30 @@ varnish.MGT.child_exit,gauge,,process,second,Child processes the were cleanly st
 varnish.MGT.child_panic,gauge,,process,second,Child processes that panicked. This metric is only provided by varnish >=4.x.,0,varnish,mgt child panic
 varnish.MGT.child_start,gauge,,process,second,Child processes that started. This metric is only provided by varnish >=4.x.,0,varnish,mgt child start
 varnish.MGT.child_stop,gauge,,process,second,Child processes that exited with an unexpected return code. This metric is only provided by varnish >=4.x.,0,varnish,mgt child stop
-varnish.MGT.uptime,gauge,,second,,This metric is only provided by varnish >=4.x.,0,varnish,mgt uptime
+varnish.MGT.uptime,gauge,,second,second,Management process uptime. This metric is only provided by varnish >=4.x.,0,varnish,mgt uptime
```

Comment:
```suggestion
varnish.MGT.uptime,gauge,,second,,Management process uptime. This metric is only provided by varnish >=4.x.,0,varnish,mgt uptime
```
Should it be `second` instead of `second/second` ?

Reply:
Good catch, it's a emitted as a counter (in second) by varnish. That means the integration computes the rate of it and sends that as the uptime in second/second. Likely something to be improved but not something changed by this PR

Reply:
It is submitted as a rate so I guess it should be `second,second`